.gitmodules

```diff
@@ -16,7 +16,7 @@
 [submodule "BaseTools/Source/C/BrotliCompress/brotli"]
 	path = BaseTools/Source/C/BrotliCompress/brotli
 	url = https://github.com/google/brotli
-	ignore = untracked
+	ignore = untracked
```

Comment:
AIUI you are fixing the whitespace here to be consistent throughout the file? That is appreciated, but please do it as a separate patch.